dll/win32/browseui/settings.cpp

```diff
@@ -0,0 +1,38 @@
+/*
+ * PROJECT:     ReactOS Explorer
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
```

Comment:
```suggestion
 * PROJECT:     ReactOS browseui
 * LICENSE:     LGPL-2.1-or-later (https://spdx.org/licenses/LGPL-2.1-or-later)
```

Reply:
Done.

Reply:
Please keep it lowercase as in my suggestion, see:

- https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=grep&s=PROJECT%3A+++++ReactOS+browseui
- https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=grep&s=PROJECT%3A+++++ReactOS+BrowseUI

Reply:
Changed to lowercase